internal/crypto/src/ec_utils.rs

```diff
@@ -127,6 +127,7 @@ pub(crate) fn der_to_p1363(data: &[u8], sig_len: usize) -> Result<Vec<u8>, RawSi
 }
 
 // Returns supported EcdsaCurve for given public key.
+#[allow(dead_code)]
```

Comment:
Do we need either of these utilities now? Will they serve a purpose in the future or should we just remove them?

Reply:
Yes, it used by Verify for EC certs that have different curves.  The dead code is because of the feature flags.